delta/plugins/elasticsearch/src/test/resources/akka-persistence-test.conf

```diff
@@ -0,0 +1,42 @@
+akka {
+  persistence {
+    journal.plugin = "jdbc-journal"
+    snapshot-store.plugin = "jdbc-snapshot-store"
+  }
+}
+
+jdbc-journal {
+  slick = ${h2}
+}
+
+jdbc-snapshot-store {
+  slick = ${h2}
+}
+
+jdbc-read-journal {
+  includeLogicallyDeleted = ${akka-persistence-jdbc.logicalDeletion.enable}
+  refresh-interval = "10ms"
+  max-buffer-size = "500"
+  slick = ${h2}
+}
+
+akka-kryo-serialization {
+  id-strategy = "automatic"
+  implicit-registration-logging = true
+  resolve-subclasses = false
+}
+
+h2 {
+  profile = "slick.jdbc.H2Profile$"
+  db {
+    url = "jdbc:h2:mem:"${test-database}";DATABASE_TO_UPPER=false;"
```

Comment:
I changed test-database to test-instance in my last PR to create a custom cache directory